le_lambda.py

```diff
@@ -1,75 +1,56 @@
+import logging
 import boto3
 import socket
 import ssl
-import datetime
 import re
 import urllib
 import csv
 import zlib
 import json
 import certifi
-from le_config import *
+import os
+from uuid import UUID
 
-print('Loading function')
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+logger.info('Loading function...')
 
 s3 = boto3.client('s3')
 
+R7_REGION = os.environ.get('r7_region')
+ENDPOINT = 'data.{}.logentries.com'.format(R7_REGION)
+PORT = 20000
+LOG_TOKEN = os.environ.get('log_token')
+
 
 def lambda_handler(event, context):
-    host = 'data.logentries.com'
-    port = 443
-    s_ = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    s = ssl.wrap_socket(
-                sock=s_,
-                keyfile=None,
-                certfile=None,
-                server_side=False,
-                cert_reqs=ssl.CERT_REQUIRED,
-                ssl_version=getattr(
-                    ssl,
-                    'PROTOCOL_TLSv1_2',
-                    ssl.PROTOCOL_TLSv1
-                ),
-                ca_certs=certifi.where(),
-                do_handshake_on_connect=True,
-                suppress_ragged_eofs=True,
-            )
-    s.connect((host, port))
-    tokens = []
-    if validate_uuid(debug_token) is True:
-        tokens.append(debug_token)
-    if validate_uuid(lambda_token) is True:
-        tokens.append(lambda_token)
-    else:
-        pass
+    sock = create_socket()
 
-    if validate_uuid(log_token) is False:
-        for token in tokens:
-            s.sendall('%s %s\n' % (token, "{}: log token not present for username={}"
-                                   .format(str(datetime.datetime.utcnow()), username)))
+    if validate_uuid(LOG_TOKEN) is False:
+        logger.critical('{} is not a valid token. Exiting.'.format(LOG_TOKEN))
```

Comment:
Instead of explicit string formatting this logger usage is a bit clearer I think:

`logger.critical('%s is not a valid token. Exiting.', LOG_TOKEN)`